CHANGELOG.md

```diff
@@ -5,6 +5,83 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [0.43.0] - 2025-03-20
+#### ✨ Highlights
+
+- 🚀 Added support for [shortcuts](https://pixi.sh/v0.43.0/global_tools/introduction/#shortcuts) in `pixi global`.
```

Comment:
I think for past release notes we start more with a sentence that describes what was changed. DO you think we can do it also for this highlight? ( or if it's worth )